src/wh_client_crypto.c

```diff
@@ -791,6 +1070,58 @@ printf("[client] %s ctx:%p key:%p, sig:%p sig_len:%u, hash:%p hash_len:%u out_re
     return ret;
 }
 
+#if 0
```

Comment:
are we going to leave this in for the release or is it pending?

Reply:
Leave it for now since I think it is working, but there's now way to directly exercise it until I update wc.